lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -419,6 +419,10 @@ class ArynPartitioner(Partitioner):
              either pdfminer or OCR. Currently supports the 'object_type' property for pdfminer,
              which can be set to 'boxes' or 'lines' to control the granularity of output.
         source: The application that is using the partitioner. This is used for logging purposes.
+        output_label_options: A dictionary for configuring output label behavior. It supports two options: 
+        promote_title, a boolean that specifies whether to add a title to partitioned elements if one is missing, and
+            title_candidate_elements, a list of strings representing labels for potential titles.
+            default: {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
```

Comment:
The default specified in this docstring is wrong. The default for promote_title is False, not True.
https://github.com/aryn-ai/sycamore/blob/1f053475507aa249d3aa839e93e3a3f67027c3c8/lib/sycamore/sycamore/transforms/detr_partitioner.py#L205

Reply:
This has been fixed in #1057